docs/layouts/shortcodes/guides/config-desc.html

```diff
@@ -14,7 +14,7 @@
 
 <div class="highlight">
 <pre class="chroma"><code class="language-bash" data-lang="bash"><span class="c1"># Create empty Platform.sh configuration files</span>
-$ touch .platform.app.yaml && mkdir .platform && touch .platform/routes.yaml && touch .platform/services.yaml</code></pre>
+$ touch .platform.app.yaml && touch .platform/routes.yaml && touch .platform/services.yaml</code></pre>
```

Comment:
Why would `.platform` already exist?  That doesn't make sense to me.

Reply:
- If you clone from an initialized template, it's gonna have .platform
- When we give them the steps to create an empty project, we give them set-remote, which will create .platform
- So integration was the odd man out here.

Reply:
* The guides say if you are using a template then all of this stuff is there already, just read through for understanding. So that's not relevant.
* Wait, `set-remote` creates `.platform`? ... For the local dir, right.  Blargh.  Is there a mkdir alternative that doesn't fail if the directory already exists?  That would be the better solution on that one.

Reply:
Good point. `mkdir -p .platform`

Reply:
Let's do that, then.